packages/p2p/src/components/my-ads/quick-add-modal.jsx

```diff
@@ -311,10 +319,13 @@ const QuickAddModal = ({ advert }) => {
         return (
             <Modal
                 className='p2p-my-ads__modal-error'
-                has_close_icon={false}
+                has_close_icon
                 height='660px'
                 is_open={my_ads_store.is_quick_add_modal_open}
                 title={localize('Add payment method')}
+                toggleModal={e => {
+                    if (!e.target || e.target.className !== 'dc-dropdown-list__item') setShouldCloseAllModals(true);
```

Comment:
There was an issue with the `Modal` component detecting the `Autocomplete` dropdown component, it closes the modal when I clicked on one of the dropdown item even though I was not clicking outside the modal (The `toggleModal` function is being called from `useOnClickOutside` hook in the `Modal` component, this condition is to ignore the dropdown item component and avoid closing the modal when user clicks on it.